src/config/selectors.tsx

```diff
@@ -42,7 +42,10 @@ export const buildMultipleChoicesButtonCy = (
 ) => `multipleChoicesButton-${idx}-${isSelected}`;
 export const PLAY_VIEW_SUBMIT_BUTTON_CY = 'playViewSubmitButton';
 export const PLAY_VIEW_SLIDER_CY = 'playViewSlider';
-export const PLAY_VIEW_TEXT_INPUT_CY = 'playViewTextInput';
+export const buildPlayViewTextInputCy = (isCorrect?: boolean) =>
+  `playViewTextInput-${
+    isCorrect === undefined ? 'undefined' : isCorrect ? 'true' : 'false'
```

Comment:
1. it is strange that you did not get a lint warning for using nested ternary expressions
2. I think you could just use the string template with the value, as it will translate the values by itself:
<img width="383" alt="Screenshot 2024-01-15 at 08 52 01" src="https://github.com/graasp/graasp-app-quiz/assets/39373170/dc2de396-a5ef-4d90-9ef9-0f0c3236698e">